INSTALL.md

```diff
@@ -54,5 +67,41 @@ To start the wubloader, simply run
 To stop the wubloader and clean up, simply run
 
   `docker-compose down`
-  
-To backfill from a node, the other nodes need to know about it. The best way to do this is to add the node to the database's nodes table.
+
+## Database setup
+
+When setting up a database node, a number of database specific options can be set.
+
+* `database_path`, the local path to save the database to. If this directory is empty then the database setups scripts will be run to create a new database. Otherwise, the database container will load the database stored in this folder.
+* `db_args.user`, `db_args.password`, the username and password for the database user that the rest of the wubloader will connect to.
+* `db_super_user`, `super_password`, the username and password for the database superuser that is only accessible from the local machine. 
+* `db_replication_user`, `db_replication_password`, the username and password for the database user other nodes can connect as to replicate the database. If `db_replication_user` is an empty string, remote replication will be disabled.
+* `db_standby`, If true this database node will replicate the database node given by `db_args.host`. 
+
+It is recommended that the passwords be changed from the defaults in production.
+A database node needs to expose its database on a port. By default this is `5432` but the port exposed to the outside can be changed in the `ports` object.
+
+The `events` table will be automatically populated by the `sheetsync`. The startup script will attempt to populated the `nodes` and `editors` tables from the `nodes.csv` and `editors.csv` files in `segments_path` directory. The expected format for these files is:
```

Comment:
What's the behaviour if these tables already exist and are populated, and the contents differ? Are they replaced? Merged (ie. anything new is added but otherwise unchanged)?